tests/test_gui_rtdc.py

```diff
@@ -0,0 +1,88 @@
+from PyQt6 import QtCore, QtWidgets
+import pathlib
+import tempfile as tf
+import shutil
+
+from mpl_data_cast.gui.main import MPLDataCast
+from helper import retrieve_data
+
+data_path = pathlib.Path(__file__).resolve().parent / "data"
+
+
+def test_setting_paths(qtbot, tmp_path):
+    """Check that setting the input and output paths works."""
+    mpldc = MPLDataCast()
+    path_in = retrieve_data("rcp_rtdc_mask-contour_2018.zip")
+    path_out = tmp_path
+    mpldc.widget_input.lineEdit_input.setText(str(path_in))
+    mpldc.widget_output.lineEdit_output.setText(str(path_out))
+
+
+def test_transfer_data_simple(qtbot, monkeypatch):
+    """Check that data is transfered when clicking on the button 'Transfer'."""
+    mpldc = MPLDataCast()
+    qtbot.add_widget(mpldc)
+    path_in = data_path / "example_dir"
```

Comment:
Please only put .zip files in the "data" directory and don't add directory structures (example_dir) that you would need for the tests. You can either zip the directory structure or create the directory structure during the test.

Reply:
Okay, I removed the test data that I added and use the already existing test data.